### PR TITLE
OEUI-177 : Remove lab tests from draft orders

### PR DESCRIPTION
**Pull request by rhenshaw56**
[OEUI-177](https://issues.openmrs.org/browse/OEUI-177) -> Implemented deletion from lab draft order

***Restyled buttons to accommodate deletions and urgency***

**The urgency button is a bit off for panels and that's because it wasn't factored into the original pr which was merged that panels would in fact have urgency but this would be fixed once [OEUI-178](https://issues.openmrs.org/browse/OEUI-178) is redone.***

### CODE REVIEW

**Comment by mogoodrich** [comment]
Looks good to me!  But I'm not super familiar with the order entry code, so will wait until another pair of eyes looks at it before merging.

**Comment by Efosaok** [comment]
please fix the failing builds

File: app/js/components/labOrderEntry/LabDraftOrder.jsx [hunk 1]
```diff
@@ -32,18 +32,36 @@ export class LabDraftOrder extends PureComponent {
...
             </a>
           </span>
+          <span className="action-btn right">
+            <a
+              id="draft-discard-btn" href="#" onClick={(event) => {
+                event.preventDefault();
+                const action = !isPanel ? 'single' : 'panel';
+                handleDraftDiscard(order, action);
+              }
+              }
```
Comment (IAMOTZ) [inline app/js/components/labOrderEntry/LabDraftOrder.jsx]: Hi @rhenshaw56 
I suggest you compose this: 
```
onClick={(event) => {
     event.preventDefault();
     const action = !isPanel ? 'single' : 'panel';
     handleDraftDiscard(order, action);
}
```
in a separate function and call the function when needed.

File: app/css/openmrs-owa-orderentry.css [hunk 2]
```diff
@@ -482,39 +482,57 @@ a.button:hover {
...
+    padding: 10px 10px;
+    width: 60%;
+}
+
 .stay-right{
     position: absolute;
     right: 10px;
 }
 
+.push-right {
```
Comment (jomadoye) [inline app/css/openmrs-owa-orderentry.css]: This class does not seem to be used, and also has the `!mportant` tag, please let's remove it.

File: app/js/components/labOrderEntry/LabEntryForm.jsx [hunk 3]
```diff
@@ -151,6 +150,13 @@ export class LabEntryForm extends PureComponent {
     }
   }
 
+  discardTestInDraft = (test, action) => {
+    const { dispatch } = this.props;
+    if (action === 'single') return dispatch(removeTestFromDraft(test));
+    if (action === 'panel') return dispatch(removeTestPanelFromDraft(test));
+    return dispatch(deleteDraftLabOrder());
+  }
```
Comment (IAMOTZ) [inline app/js/components/labOrderEntry/LabEntryForm.jsx]: The variable name `discardTestInDraft` implies that this function is meant to remove a test from the draft but from the implementation details, I see that there is a possibility of this function deleting the draft itself.
I suggest you separate the two functionalities in different functions or better still you make the name of this function reflect what it really does.

File: app/js/components/labOrderEntry/LabDraftOrder.jsx (hunk 1 above)
Comment (rhenshaw56) [reply app/js/components/labOrderEntry/LabDraftOrder.jsx]: @IAMOTZ  done!

File: app/css/openmrs-owa-orderentry.css (hunk 2 above)
Comment (rhenshaw56) [reply app/css/openmrs-owa-orderentry.css]: done

File: app/js/components/labOrderEntry/LabEntryForm.jsx (hunk 3 above)
Comment (rhenshaw56) [reply app/js/components/labOrderEntry/LabEntryForm.jsx]: @IAMOTZ  done

**Comment by mogoodrich** [comment]
We've got enough LGTMs that I'm going to go ahead and merge!  :)

**Pull request merged**